docs/docs/integrations/plus.md

```diff
@@ -19,11 +19,11 @@ Once logged in, you can generate an API key for Frigate in Settings.
 
 ### Set your API key
 
-In Frigate, you can use an environment variable or a docker secret named `PLUS_API_KEY` to enable the `Frigate+` buttons on the Explore page. Home Assistant Addon users can set it under Settings > Addons > Frigate NVR > Configuration > Options (be sure to toggle the "Show unused optional configuration options" switch).
+In Frigate, you can use an environment variable or a docker secret named `PLUS_API_KEY` to enable the `Frigate+` buttons on the Explore page. Home Assistant Addon users can set it under Settings > Add-ons > Frigate > Configuration > Options (be sure to toggle the "Show unused optional configuration options" switch).
```

Comment:
I am pretty sure it is still called `Frigate NVR` in the addon list, no?

Reply:
No, it was changed to just Frigate.

Reply:
Long time ago:

- https://github.com/blakeblackshear/frigate-hass-addons/pull/76